sfm/ui/templates/ui/harvest_list.html

```diff
@@ -1,4 +1,5 @@
 {% extends 'base.html' %}
+{% load humanize %}
 {% block title %}
```

Comment:
Looks like {% load ui_extras %} is needed here. I am getting the error: 
TemplateSyntaxError at /ui/collections/5/harvests/
Invalid filter: 'join_stats'